builtin/logical/pki/fields.go

```diff
@@ -326,3 +326,15 @@ func addCAIssueFields(fields map[string]*framework.FieldSchema) map[string]*fram
 
 	return fields
 }
+
+func addNotBeforeField(fields map[string]*framework.FieldSchema) map[string]*framework.FieldSchema {
```

Comment:
I'd perhaps suggest mirroring the value in `path_roles`: 

```go
            "not_before_duration": {
                Type:        framework.TypeDurationSecond,
                Default:     30,
                Description: `The duration before now the cert needs to be created / signed.`,
                DisplayAttrs: &framework.DisplayAttributes{
                    Value: 30,
                },
```

That way it can be re-used there if it is restructured eventually.